### PR TITLE
MAINTAINERS: move @Yutyo into the proper section

### PR DESCRIPTION
**Pull request by bl-ue**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -53,8 +55,7 @@ If you are an owner of the organization, you should be able to see an automated
   [24 August 2020](https://github.com/tldr-pages/tldr/issues/4291) — [05 October 2020](https://github.com/tldr-pages/tldr/issues/4504)
 - bl-ue ([@bl-ue](https://github.com/bl-ue)):
   [01 January 2021](https://github.com/tldr-pages/tldr/issues/5056) — [02 February 2021](https://github.com/tldr-pages/tldr/issues/5219)
-- Tan Siret Akıncı ([@Yutyo](https://github.com/Yutyo)):
-  [03 March 2021](https://github.com/tldr-pages/tldr/issues/5345) — present
+
```
Comment (bl-ue) [inline MAINTAINERS.md:58]:
```suggestion
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — COMMENTED** [review]
Hrm, I don't think I understand. This just moves it from 1 place to another in the same section?

**Comment by navarroaxel** [comment]
> Hrm, I don't think I understand. This just moves it from 1 place to another in the same section?

Yep, because the collaborators without an end date come first in the list.

**Review by sbrl — APPROVED** [review]
Thanks, @bl-ue! Ah, I understand now.

**Pull request merged**